packages/filesystem/src/node/node-filesystem.ts

```diff
@@ -379,6 +379,14 @@ export class FileSystemNode implements FileSystem {
         }
     }
 
+    async getFsPath(uri: string): Promise<string | undefined> {
+        if (!uri.startsWith('file:/')) {
+            return undefined;
```

Comment:
@JanKoehnlein Is there any reason this function ignore paths which don't begin with `file:/`? This function would be useful for correctly resolving debug file paths on Windows, however they are of the form `/c:/Users/Rob/Projects/theia-workspace/test.js`

Reply:
@thegecko Could we wrap a path in URI in the debug extension? We have a contract that we pass normalized URIs between frontend and backend.

Reply:
@thegecko I find it a bit dangerous to pass non-normalized URIs. From what I understood, we're free to implement all kinds of URI schemes to retrieve / store resources, not necessarily in the file system. The assumption that URIs without a scheme are file URIs should be made in the clients of this API.

Reply:
Acknowledged

@akosyakov 
> Could we wrap a path in URI in the debug extension?

At one point I did implement it this way so this function accepts a URI or string. Happy to go back to that model and keep the `file` check in only when a string is passed in.